### PR TITLE
127 feat add toasts to unimplemented features

### PR DESCRIPTION
**Pull request by pmdlt**
This PR add Toasts for M2 when we need it.

### CODE REVIEW

**Comment by pmdlt** [comment]
Ready to be reviewed

**Review by Thechi2000 — APPROVED** [review]
Thanks, that was deeply needed !

It is not working on Linux, but it's good on Android, that's the important part.

File: lib/views/profile_view.dart [hunk 1]
```diff
@@ -43,6 +44,14 @@ class _ProfilePageState extends State<ProfilePageView> {
                 builder: (context) => const InterestWizardView()));
       default:
         debugPrint("Click on ${e.name}");
+        Fluttertoast.showToast(
+            msg: "The view for ${e.name} is not yet implemented.",
+            toastLength: Toast.LENGTH_SHORT,
+            gravity: ToastGravity.CENTER,
+            timeInSecForIosWeb: 1,
+            backgroundColor: Color.fromARGB(255, 22, 231, 105),
```
Comment (quartztz) [inline lib/views/profile_view.dart:52]: At some point we will want to have this as a theme value :))

**Review by quartztz — APPROVED** [review]
looks great! thank you so much

**Pull request merged**